google/cloud/firestore_v1/collection.py

```diff
@@ -272,6 +271,24 @@ def limit(self, count):
         query = query_mod.Query(self)
         return query.limit(count)
 
+    def limit_to_last(self, count):
+        """Create a limited to last query with this collection as parent.
+
+        See
```

Comment:
Can we add the note here about mutual exclusivity with `limit` and `limit_to_last` here too?

Reply:
Done